mirage/views/private/utils.ts

```diff
@@ -0,0 +1,259 @@
+export enum comparisonOperators {
+    eq = 'eq',
+    ne = 'ne',
+    lt = 'lt',
+    lte = 'lte',
+    gt = 'gt',
+    gte = 'gte',
+}
+
+export interface ProcessOptions {
+    defaultPageSize?: number;
+    defaultSortKey?: string;
+}
+
+interface QueryParameters {
+    [key: string]: string;
+}
+
+export interface JsonData {
+    data: any[];
+    links: {};
+    meta: {};
+}
+
+// Fields marked always_embed in the osf api code should be added to this constant
+// The key is the model and the list contains the relationships to embed.
+const alwaysEmbed: { [key: string]: string[] } = {
+    contributor: ['users'],
+};
```

Comment:
I don't disagree, although it will only be used for this purpose. We should never actually need it outside of this file, but we wouldn't necessarily think to update it here when it's time to update. Soooo. I could put it in a file at the top-level of the mirage directory?